### PR TITLE
Tvm 14 - Add preference and page settings updated

### PR DESCRIPTION
**Pull request by RaphBout1**

### CODE REVIEW

File: src/components/IconPicker/IconDisplay.jsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import { Box, Typography } from "@mui/material";
+import {
+  Home,
+  Settings,
+  Person,
+  Search,
+  Add,
+  Edit,
```
Comment (mineralsfree) [inline src/components/IconPicker/IconDisplay.jsx:9]: imho, the icons here seems to be a bit weird given the specificity of the project, I'll create a ticket to maybe consider alternative list ;)

File: src/app/settings/page.jsx [hunk 2]
```diff
@@ -1,20 +1,40 @@
...
 import SavedSearchList from '@/components/User/SavedSearchList';
 import AddNewPreferenceButton from '@/components/User/PreferenceButton';
 import { Box } from '@mui/material'; 
+import { useState, useEffect } from 'react';
 
 export default function Page() {
+  const [userData, setUserData] = useState(null);
+
+  useEffect(() => {
+    fetch('/api/users')
```
Comment (mineralsfree) [inline src/app/settings/page.jsx:13]: we fetch user in 3 different places now, I've created a ticket to fetch it in Redux and reuse, can be as is in this PR

File: src/models/User.js [hunk 3]
```diff
@@ -1,10 +1,33 @@
...
+    countryOfOrigin: {
+      type: String,
+      required: true,
+    },
+    countryCode: {
+      type: String,
+      required: true,
+    },
+    photo: {
+      type: String,
```
Comment (mineralsfree) [inline src/models/User.js:22]: What is photo? base64 string?

File: src/components/User/UserCard.jsx [hunk 4]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { Box, Typography, Avatar, Grid, Card, CardContent } from '@mui/material';
 
 const UserCard = ({ user }) => {
+  console.log(user);
```
Comment (mineralsfree) [inline src/components/User/UserCard.jsx:5]: please, remove :)

**Review by mineralsfree — APPROVED** [review]

File: src/components/IconPicker/IconDisplay.jsx (hunk 1 above)
Comment (RaphBout1) [reply src/components/IconPicker/IconDisplay.jsx:9]: yes it's just a list random

File: src/models/User.js (hunk 3 above)
Comment (RaphBout1) [reply src/models/User.js:22]: It's just a link to an image online it's not a real feature

**Pull request merged**